### DISCUSSION TITLE
显示账号被冻结

### DISCUSSION THREAD

**Discussion by 213366** [Richasy/Bili.Uwp#650] [问答]
在登录时二维码扫描无效（手上一个vivo和ipad都扫不出来）使用密码登陆提示账号被冻结。
但在网页端可以正常登录。

软件arm64 版
设备华为Matebooke2019（骁龙850）
系统windows11 22000.348

**Comment by Sharkser** [comment]
同显示账号被冻结，浏览器手机登陆都没问题

**Comment by tjx4135** [comment]
同样显示账号被冻结中，以及无效的二维码。

**Comment by GoGiladam** [comment]
我也遇到了这个问题，因为我用的play版bilibili，用国内版本bilibili，再扫码就可以登陆了

**Comment by juzijun233** [comment]
<img width="325" alt="image" src="https://user-images.githubusercontent.com/43207394/147397060-f0090e98-d037-45fc-935c-ac3ed052f573.png">
密码无法登录
扫码（手机App）无反应，正式版和概念版都不行

**Reply by Q-Tai-mu** [reply]
最近登录接口要换下吧
